letsencrypt/client.py

```diff
@@ -138,22 +138,22 @@ def obtain_certificate(self, domains, csr=None):
 
         # Save Certificate
         cert_path, chain_path = self.save_certificate(
-            certr, self.config.cert_path, self.config.chain_path)
+            certr, self.config.cert_dir, self.config.cert_dir)
 
         revoker.Revoker.store_cert_key(
             cert_path, self.account.key.file, self.config)
 
         return cert_key, cert_path, chain_path
 
-    def save_certificate(self, certr, cert_path, chain_path):
+    def save_certificate(self, certr, cert_dir, chain_dir):
```

Comment:
`le_util.unique_file` take a path to a file, not directory, so I think all changes to `save_certificate` should be reverted